src/client/img/buttonPlusSvg.tsx

```diff
@@ -0,0 +1,15 @@
+import { FunctionComponent } from 'react';
+
+export const ButtonPlusSvg: FunctionComponent<React.ComponentProps<'svg'>> = (props) => (
+  <svg width="15" height="16" viewBox="0 0 15 16" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
```

Comment:
1. Why this svg is so strange?
2. Why `Image` tag is not sufficient for this case?

Reply:
I know, here: 
https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/pull/97#discussion_r1370757516

Reply:
Oh I see.
So, we can keep Icons as components, to add [@svgr/webpack](https://www.npmjs.com/package/@svgr/webpack) plugin and import them as a component.

But I would like to offer you another approach for icon coloring. 

We can use [currentColor](https://css-tricks.com/currentcolor/) value inside svg, for example `fill="currentColor"`. Then we just need to define CSS `color` property and it will be inherited by svg.

Reply:
Wow I had no idea `currentColor` worked that, but also I can't make it work? I am setting `color`, `fill`, `stroke` on both the SVG path and the wrapping HTML element but the color does not change. Even if I made it work somehow, this feels like an unusual/unnecessary gotcha to leave for someone in the future to figure out. I think I would prefer to just import the SVGs as components using `svgr` or `next-images`. The advantages of `Image` don't really seem to matter for super simple vector icons like this, what do you think?

Reply:
The problem with `svgr` is that it breaks our current workflow if importing SVGs and using them as `Image.src`. `next-images` seems to be meant for usage with `img`, that's also not what we want. By process of elimination I arrived back at my current solution as the best alternative for now?

Reply:
Let's continue with a current approach, but with a `currentColor` for styling svg?